wercker.yml

```diff
@@ -9,16 +9,16 @@ build:
         code: ./build
 
     - script:
-        name: copy binary
-        code: cp bin/linux_amd64/farva-gateway "$WERCKER_OUTPUT_DIR"
+        name: copy binaries
+        code: cp bin/linux_amd64/* "$WERCKER_OUTPUT_DIR"
 
 deploy:
     box: nginx:1.10.0
     steps:
     - internal/docker-push:
         username: $QUAY_USERNAME
         password: $QUAY_PASSWORD
-        cmd: /pipeline/source/farva-gateway
+        working-dir: /pipeline/source/
```

Comment:
You must provide a 'command' directive in the pod now. The other two options I see are to ship a single binary that takes an arg (e.g. `farva monitor` or `farva gateway`) or to ship two different images. Which of the three do you prefer?